### DISCUSSION TITLE
某些牌谱解析会报错

### DISCUSSION THREAD

**Discussion by shinnchouu** [Equim-chan/mjai-reviewer#149] [General]
如题
我的牌谱地址如下: https://game.maj-soul.com/1/?paipu=240821-01278283-d6c0-4a9c-b99d-2d11a11d2052_a50606868
报错信息如下:
An error occurred during the task, please check your inputs.

Common causes:
* Rule violation or ruleset is not compatible
* A type of tile is seen for more than 4 times
* Dora indicator is not set
* Total tsumo count overflows the capacity (70)

**Comment by Equim-chan** [comment]
感谢反馈，现在已经修复了。
